CHANGELOG.md

```diff
@@ -8,19 +8,48 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 <!--
 ## Unreleased
 
+### Fixed
+
+### Added
+
+### Changed
+
+-->
+## v1.1.0
 ### Fixed
 - Autofill for empty slug
+- Fixed Web content preview in CMS
+- Fixed related pages export
+- Fixed API tests
 
 ### Added
+- Added support for SMS content
+- Added support for USSD content
+- Added support for a Footer in WhatsApp content
+- Added support for List Messages in WhatsApp content
+- Added error handling on WhatsApp template submission errors, and adds issue to sentry
+- Added validation of variables used in WhatsApp template submission
+- Added WhatsApp title check
+- Added typing information for migration test
+- Added CI checks for pending migrations
+- Fixed Redis caching issues
 
 ### Changed
 - Moved slug uniqueness validation to the model validation/clean
 - Empty slugs will auto-generate a unique slug, but if a duplicate slug is specified the user will get a validation error instead of their chosen slug getting overwritten with a unique one.
 - Slug uniqueness is per-locale
 - Test speedups. Tests now run in parallel by default, and there's a separate contentrepo/settings/test.py for test-specific settings.
 - Tests no longer run in parallel by default, because the output is a little less clear and the speedup is negligible on some systems.
+- WhatsApp example values no longer show on the default API view, and forms part of the fields returned when the parameter `whatsapp=true` is added to the api call
+- Improved testing of Importer
```

Comment:
Maybe add the new importer to this? We mention removing the old one, but not adding the new one.

Reply:
I think the importer was added in the dev releases prior to this, but let me know if you strongly disagree?

Reply:
Oh, right. I missed the dev release sections.

It's fairly common for non-dev release changelogs to include all the relevant changes from the dev releases leading up to them, so people don't have to read a bunch of dev/beta/rc changelogs as well. I think not doing that here is fine, given that it would add a bunch of extra work and we're not *really* stable yet, but that's probably something for @rudigiesler to decide. 🤷